stdlib/public/core/Codable.swift

```diff
@@ -67,6 +67,12 @@ public protocol CodingKey {
     init?(intValue: Int)
 }
 
+extension CodingKey {
+    public var description: String {
+        return stringValue
```

Comment:
Improved description in 33582e6 and 98ed5f9, e.g. `MyCodingKey(stringValue: "someKey", intValue: nil)`